outlines/models/transformers.py

```diff
@@ -456,33 +549,34 @@ def __init__(self, model: "PreTrainedModel", processor):
 
         super().__init__(model, tokenizer)
 
-        self.type_adapter = TransformersMultiModalTypeAdapter()
+        self.type_adapter = TransformersMultiModalTypeAdapter(
```

Comment:
Is there a reason to prefer instantiating the type adapter with a tokenizer instead of using `apply_chat_template` from within this class?

Reply:
The reason is just to try to contain as much user input formatting logic as possible in the type adapter to follow the general pattern.

Reply:
That's a good reason